I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/chapter01.rpy", line 124, in script
    jump day_2
ScriptError: could not find label 'day_2'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/chapter01.rpy", line 124, in script
    jump day_2
  File "C:\Users\prans\Downloads\Setups\renpy-7.4.11-sdk.7z\renpy-7.4.11-sdk/renpy/ast.py", line 1757, in execute
    rv = renpy.game.script.lookup(target)
  File "C:\Users\prans\Downloads\Setups\renpy-7.4.11-sdk.7z\renpy-7.4.11-sdk/renpy/script.py", line 918, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'day_2'.

Windows-10-10.0.19041
Ren'Py 7.4.11.2266
donor 1.0
Thu Jan 20 14:31:43 2022
